### PR TITLE
Added tabs and activities

### PR DESCRIPTION
**Pull request by mwyoung**
I was able to create tabs that work, along with a (very) simple activity for each Pokemon. I also found a URL that gives larger artwork for each Pokemon (and is **very** hacky in my current method). 
This issue should be merged after Trevor completes #6, as to make the tabs work required moving lots of code from the MainActivity to a new class.
Let me know if I need to change/add anything.

### CODE REVIEW

**Comment by hammockt** [comment]
This looks dope.

I also agree that #6 should probably be merged before this. Honestly that pull request is finished, I am just waiting to see if @magical has any commentary on it. Especially with the nonsense of:
```java
MutableLiveData<HashMap<Integer, LiveData<Pokemon>>>
```

Also I have pretty much finished the caching code on some other branches. I will make pull requests for those latter because they depend on another. I think this should be merged in before that, especially before the last pull request in that series (it unfortunately has to touch everything).

**Review by hammockt — APPROVED** [review]

**Comment by mwyoung** [comment]
I have also added a refresh button on the main page, and a share and open in Bulbapedia button (requires name, not yet implemented) in the Pokemon details (as per assignment description).

**Comment by mwyoung** [comment]
I have merged Trevor's changes from #6, and it works. I also had to push all of the functions in PokeAPIViewModel() to another function that I could call to make the refresh button work. If I should remove this let me know.

**Comment by hammockt** [comment]
LGTM. I don't know if the refresh will survive the eventual caching merge, but I will make an honest effort to try. If you can fix the branch the conflicts we can probably merge this in

**Review by hammockt — APPROVED** [review]

File: app/src/main/java/rec/games/pokemon/teambuilder/PokemonItemDetailActivity.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.lang.reflect.Array;
+
+public class PokemonItemDetailActivity extends AppCompatActivity
+{
+	private static final String TAG = PokemonItemDetailActivity.class.getSimpleName();
+
+	private int ArrayId;
```
Comment (magical) [inline app/src/main/java/rec/games/pokemon/teambuilder/PokemonItemDetailActivity.java]: It looks weird to have a variable that starts with an uppercase letter. 
Also what number is this? Is it the pokemon id?

**Review by magical — APPROVED** [review]

File: app/src/main/java/rec/games/pokemon/teambuilder/PokemonItemDetailActivity.java (hunk 1 above)
Comment (mwyoung) [reply app/src/main/java/rec/games/pokemon/teambuilder/PokemonItemDetailActivity.java]: That is just a temp variable that I get from the array position, mostly to be able to show the art image. I will change the variable name to make it more apparent that it needs to be replaced at a future date.

**Pull request merged**